libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/app/MobileSyncSDKManager.java

```diff
@@ -95,6 +100,7 @@ public static void initNative(Context context, Class<? extends Activity> mainAct
      * @param context       Application context.
      * @param mainActivity  Activity that should be launched after the login flow.
      * @param loginActivity Login activity.
+     * @noinspection unused
```

Comment:
Since the older Java code does compile with a lot of warnings now, I did make an effort to resolve trivial warnings in any source that was already impacted by this pull request.  I did not attempt to resolve any warning the would impose a logic or feature change.  I didn't want to make the diff to unwieldy or add more risk of regression.